package.json

```diff
@@ -86,5 +86,8 @@
   "packageManager": "yarn@3.6.3",
   "publishConfig": {
     "access": "public"
+  },
+  "resolutions": {
+    "@lifi/types": "portal:/Users/kristofferhovlandberg/github/lifi/types"
```

Comment:
Please don't use local version here 🙂

Reply:
it works on my computer 🤔

Reply:
no jk have released major of types now and included it